docs/features/configurations.md

```diff
@@ -0,0 +1,141 @@
+# Configurations
+Atmosphère provides a variety of customizable configurations to better adjust to users' needs.
+
+## BCT.ini
+This is the configuration file used by Fusée.
+This file is located under the `/atmosphere/config/` folder on your SD card and a default template can be found inside the `/atmosphere/config_templates/` folder.
+
+### Adding a Custom Boot Splashscreen
+Add the following lines to BCT.ini and change the value of `custom_splash` to the actual path and filename of your boot splashscreen:
+```
+[stage2]
+custom_splash = /path/to/your/bootlogo.bmp
+```
+
+The boot splashscreen must be a BMP file, it must be 720x1280 (1280x720 rotated 90 degrees left/counterclockwise/anti-clockwise) resolution, and be in 32-bit ARGB format. You can use image editing software such as GIMP or Photoshop to export the image in this format.
+
+### Configuring "nogc" Protection
+"nogc" is a feature provided by Fusée-secondary which disables the Nintendo Switch's Game Card reader. Its purpose is to prevent the reader from being updated when the console has been updated, without burning fuses, from a lower firmware version. More specifically, from firmware versions 4.0.0 or 9.0.0 which introduced updates to the Game Card reader's firmware. By default, Atmosphère will protect the Game Card reader automatically, but you are free to change it.
+
+To change its functionality, add the following line to the `stratosphere` section and change the value of `X` according to the following list:
```

Comment:
Formatting here is inconsistent with "Adding a Custom Boot Splashscreen" w.r.t. whether the class is included in the code block or not. I would include the class in the code block to make it harder to miss.